source/usage-examples/command.txt

```diff
@@ -0,0 +1,31 @@
+=============
+Run A Command
+=============
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+You can run all :manual:`non CRUD </crud/>` database operations using the 
+:node-api:`command() </Admin.html#~command>` method. Typically ``command()`` is used for diagnostic and administrative tasks, such as fetching server stats or initializing a replica set. 
+
+.. note::
+    Using :manual:`collection methods </reference/method/js-collection>`
+    instead of raw database commands through the ``command()`` is
+    recommended for beginner users, since ``collection methods`` are
+    wrappers that abstract some configuration. 
```

Comment:
Can you clarify this? I'm not sure this is coming across so clearly.

Reply:
updated to "Use collection methods instead of raw db commands whenever possible."